util_execute_cycle.lua

```diff
@@ -102,6 +102,20 @@ local function check_digtron_size(layout)
 	end
 end
 
+-- add_pos was introduced on Jan5 and this feature on Jan 17
```

Comment:
Please mention the year too. I recommend using YYYY-MM-DD.

Reply:
I believe that version numbers are more helpful than dates in this case, for example when checking against compatibility of the mod with slow-updating repositories (Debian stable)

Suggestion: `:add_pos(...) is available since Minetest 5.9.0-dev, commit d0753ddd`

Reply:
Solved in 6cb0a32

I added both date in YYYY-MM-DD and commit.